probe-rs/Cargo.toml

```diff
@@ -201,6 +201,7 @@ clap = { version = "4.4", features = ["derive"] }
 itm = { version = "0.9.0-rc.1", default-features = false }
 insta = {version = "1.34.0", features = ["yaml", "filters"]}
 pretty_assertions = "1.4.0"
+test-case = "3.2.1"
```

Comment:
We should always use as little version digits as possible, so just "3" so cargo update will properly update this :)

Reply:
There are triple digit versions everywhere in probe-rs, so I was just following the "apparent" standard. I will update as you suggest.